app/src/main/java/org/android/go/sopt/Home/HomeViewModel.kt

```diff
@@ -0,0 +1,39 @@
+package org.android.go.sopt.Home
+
+import android.util.Log
+import android.widget.Toast
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import org.android.go.sopt.Adapter.ItemAdapter
+import org.android.go.sopt.ServicePool
+import org.android.go.sopt.data.ResponseListUsersDto
+import retrofit2.Call
+import retrofit2.Response
+
+class HomeViewModel : ViewModel() {
+    private val _userListResult: MutableLiveData<ResponseListUsersDto> = MutableLiveData()
+    val userListResult: LiveData<ResponseListUsersDto> = _userListResult
+
+    private val getListUsersService = ServicePool.GetListUsersService
+    fun completeGetUsers() {
+        getListUsersService.getusers().enqueue(object : retrofit2.Callback<ResponseListUsersDto> {
```

Comment:
getListUsersService로 따로 만들어주지 않고 바로 completeGetUsers 안에서 사용해도 좋을 것 같아용
```suggestion
          ServicePool.GetListUsersService.getusers().enqueue(object : retrofit2.Callback<ResponseListUsersDto> {
```